README.md

```diff
@@ -1,43 +1,152 @@
-## mcmicro-nf: Nextflow prototype for mcmicro
+# mcmicro-nf: Multiple-choice microscopy pipeline
+
+## Installation
 
 Install [Nextflow](https://www.nextflow.io/): `curl -s https://get.nextflow.io | bash`
 
-### Local execution
+### Additional steps for local installation
+* Install [Docker](https://docs.docker.com/install/). Ensure that the Docker engine is running by typing `docker images`. If the engine is running, it should return a (possibly empty) list of container images currently downloaded to your system.
+* (Optional) If working with TMAs, you will need MATLAB 2018a or later. Additionally, you will need to install Coreograph locally by running `nextflow run labsyspharm/mcmicro-nf/setup.nf`.
+
+## Exemplar data
+
+Two exemplars are currently available for demonstration purposes. These are
+
+* `exemplar-001` is meant to serve as a minimal reproducbile example for running all modules of the pipeline, except the dearray step. The exemplar consists of six tile images, collected on a Lung Adenocarcinoma specimen in three cycles, for a total of 12 channels. Because the exemplar is small, illumination profiles were precomputed from the larger context and included with the raw images.
+
+* `exemplar-002` is a two-by-two cut-out from a tissue microarray (TMA). The four cores are two meningioma, one GI stroma tumor, and one normal colon specimens. The exemplar is meant to test the dearray step, followed by processing of all four cores in parallel.
```

Comment:
* If you adopt my wording above just say "TMA" since it will already have been defined.
* Possibly say "specimen cores" or "cores (specimens)" to help out people who don't know what a TMA is.
* Could say "meningioma tumor" as "two meningioma" reads oddly, or maybe "meningiomas".
* Make "colon specimens" singular.